### ISSUE TITLE
The system cannot find the path specified.

### ISSUE THREAD

**Issue by josheligoldman** [3b1b/manim#173]
When I tried to export the ShowPlan scene from the nn folder in part2.py, I get the error:
The system cannot find the path specified. Then when I open ShowPlan.mp4, it's just a plain black video

C:\Users\joshe\Downloads\manim-master>C:/Python27/python.exe extract_scene.py part2 ShowPlanWriting to C:\Users\joshe\Downloads\manim-master\Exports\animations\part2\1080p60\ShowPlanTemp.mp4
Writing "\centering Plan" to C:\Users\joshe\Downloads\manim-master\files\Tex\-1352845359.texWriting "\centering $\cdot$ Recap" to C:\Users\joshe\Downloads\manim-master\files\Tex\692448994.tex
This is METAFONT, Version 2.7182818 (MiKTeX 2.9.6630 64-bit)

(C:\Users\joshe\AppData\Local\Programs\MiKTeX 2.9\fonts\source\public\rsfs\rsfs
10.mf
(C:\Users\joshe\AppData\Local\Programs\MiKTeX 2.9\fonts\source\public\cm\cmbase
.mf)
(C:\Users\joshe\AppData\Local\Programs\MiKTeX 2.9\fonts\source\public\rsfs\scri
pt.mf
(C:\Users\joshe\AppData\Local\Programs\MiKTeX 2.9\fonts\source\public\rsfs\scri
ptu.mf [65] [66] [67] [68] [69] [70] [71] [72] [73] [74] [75] [76] [77]
[78] [79] [80] [81] [82] [83] [84] [85] [86] [87] [88] [89] [90] [127]) ) )
Font metrics written on rsfs10.tfm.
Output written oThis is METAFONT, Version 2.7182818 (MiKTeX 2.9.6630 64-bit)

(C:\Users\joshe\AppData\Local\Programs\MiKTeX 2.9\fonts\source\public\rsfs\rsfs
7.mf
(C:\Users\joshe\AppData\Local\Programs\MiKTeX 2.9\fonts\source\public\cm\cmbase
.mf)
(C:\Users\joshe\AppData\Local\Programs\MiKTeX 2.9\fonts\source\public\rsfs\scri
pt.mf
(C:\Users\joshe\AppData\Local\Programs\MiKTeX 2.9\fonts\source\public\rsfs\scri
ptu.mf [65] [66] [67] [68] [69] [70] [71] [72] [73] [74] [75] [76] [77]
[78] [79] [80] [81] [82] [83] [84] [85] [86] [87] [88] [89] [90] [127]) ) )
Font metrics written on rsfs7.tfm.
Output written on rsfs7.600gf (27 characters, 6624 bytes).
Transcript written on rsfs7.log.
This is METAFONT, Version 2.7182818 (MiKTeX 2.9.6630 64-bit)

(C:\Users\joshe\AppWriting "\centering $\cdot$ Gradient descent" to C:\Users\joshe\Downloads\manim-master\files\Tex\1565636788.tex
Writing "\centering $\cdot$ Analyze this network" to C:\Users\joshe\Downloads\manim-master\files\Tex\-1964213952.tex
Writing "\centering $\cdot$ Where to learn more" to C:\Users\joshe\Downloads\manim-master\files\Tex\1692982784.tex
Writing "\centering $\cdot$ Research corner" to C:\Users\joshe\Downloads\manim-master\files\Tex\-595265111.tex
Animation 0: ShowCreationSurroundingRectangle: 100%|#########################################################################################################################################################################| 60/60 [00:05<00:00, 10.78it/s]Animation 1: FadeOutSurroundingRectangleToSurroundingRectangle: 100%|########################################################################################################################################################| 60/60 [00:05<00:00, 10.63it/s]
Animation 2: MoveToTargetVGroupToVGroup, etc.: 100%|#########################################################################################################################################################################| 60/60 [00:07<00:00,  7.91it/s]Animation 3: MoveToTargetVGroupToVGroup, etc.: 100%|#########################################################################################################################################################################| 60/60 [00:07<00:00,  7.87it/s]
Animation 4: MoveToTargetVGroupToVGroup, etc.: 100%|#########################################################################################################################################################################| 60/60 [00:07<00:00,  7.67it/s]Animation 5: MoveToTargetVGroupToVGroup, etc.: 100%|#########################################################################################################################################################################| 60/60 [00:07<00:00,  7.65it/s]
Played a total of 6 animations
The system cannot find the path specified.

**Comment by MedNait** [comment]
I had the same issue on __Windows__ (i.e. "The system cannot find the path specified" and the black screen issue). Now I am able to run `python extract_scene.py example_scenes.py SquareToCircle -l` without problems.  
Here is what I did (which again is specific to __Windows__ OS). 

* I re-downloaded the latest manim project files, then I installed the packages inside the "requirements.txt" file making sure to install the indicated versions (except for scipy for which I installed the latest version after I had problems installing the indicated version in the file and aggdraw for which the link didn't work). For aggdraw I went to the given link in the README.md file `https://github.com/scottopell/aggdraw-64bits/`  and installed this specific version  (apparently latest versions don't work with manim and I suspect that this is the reason for the __black screen issue__) without worrying about freetype.

* I modified few lines of code in `extract_scene.py`  inside function `handle_scene`:
1. Added the following:
```python
elif (platform.system() == "Windows"): 
    commands = ["start"]
```
after (around line 147)  `commands = ["xdg-open"]`.

 2. Changed the following (around line 157):
```python
sp.call(commands, stdout=FNULL, stderr=sp.STDOUT)
```
to
```python
sp.call(commands, stdout=FNULL, stderr=sp.STDOUT, shell=True)
```
* I replaced the following (around line 16) from `utils\sounds.py`
```python
"> /dev/null"
```
with
```python
"> /null"
```
which tells the `play` command to avoid outputting unnecessary data.
